docs/perfomance-analysis.md

```diff
@@ -0,0 +1,31 @@
+# Analyze a rules performance with the [Visual Studio performance profiler](https://learn.microsoft.com/en-us/visualstudio/profiling)
```

Comment:
I wouldn't put this in the repo itself. Users don't need to do this and it's hard to maintain and update.

Make it an xtranet page or google doc in the shared drive. I'd propose to sync with @costin-zaharia-sonarsource as he might have an idea how to organize some "Performance" folder there.

Reply:
I would move this to xtranet. 

Additionally, this is not enough, this approach is very useful to isolate local problems but it's not precise enough for normal execution during build. For sure we can start with this and then extend for measuring the full build.

Reply:
> For sure we can start with this and then extend for measuring the full build.

This is for tracking down problems and not so much for identifying them.

Reply:
We do have this tips and tricks page on xtranet. You can add it there: https://xtranet-sonarsource.atlassian.net/wiki/spaces/LANG/pages/2690089021/Tips+Tricks+-+.NET+Squad

Reply:
I copied the content to https://xtranet-sonarsource.atlassian.net/wiki/spaces/LANG/pages/2788524068/Analyze+a+rules+performance+with+the+Visual+Studio+performance+profiler